src/hltypes.rs

```diff
@@ -72,6 +72,17 @@ construct_tag! {
 
 impl_from_trait!(PasswordHash, 128);
 
+construct_tag! {
+	/// A type to represent the `Tag` output by Blake2b hash in keyed mode.
+	///
+	/// # Errors:
+	/// An error will be returned if:
+	/// - `slice` is not 16 bytes.
+	(Blake2bTag, test_blake2b_tag, 16, 16)
```

Comment:
1) It should be named `Tag` since there should only be one tag used in high-level, that is in `auth`.
2) Please change the lower and upper bounds to be `32` and reflect the change in documentation, since we want to use `BLAKE2b-256`.